test/specs/modules/Dropdown/Dropdown.spec.js

```diff
@@ -33,6 +37,61 @@ describe('Dropdown', () => {
     expect(openSpy).to.have.been.calledWith(false);
   });
 
+  it('should close the menu when clicking on option', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+    wrapper.vm.setOpen(true);
+    wrapper.find(DropdownItem).trigger('click');
+    wrapper.vm.$nextTick(() => {
+      expect(wrapper.vm.open).to.equal(false);
+    });
+  });
+
+  it('should close the menu when re-clicking on dropdown head', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        placeholder: 'foo',
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+    wrapper.vm.setOpen();
+
+    wrapper.find('.text').trigger('click');
+
+    wrapper.vm.$nextTick(() => {
+      expect(wrapper.vm.open).to.equal(false);
+    });
```

Comment:
Are you sure you need to use `$nextTick`? Also note that you need to make the tests asynchronous when you add a callback/promise.